src/containers/bookings/components/NoBookings/noBookingsBooked.tsx

```diff
@@ -60,8 +60,7 @@ export const NoBookingsBooked: React.FC<NoBookings> = ({ sessions }) => {
                             ${sessions
 								?.filter((session) => session.agency !== null)
 								?.map(
-									(consultant) =>
-										consultant.consultant.username
+									({ consultant }) => consultant.displayName
```

Comment:
1. We don't have always filled up the displayName on all clients. Should be `consultant.displayName || consultant.userName`

2. This is coming from the core already. So please make another pr with the change inside the core.

Reply:
Done